packages/react-scripts/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@everlong/league-react-scripts",
-  "version": "3.3.1",
+  "version": "3.3.1-pr9",
```

Comment:
I added this pr number to the version string so that we can publish our patched version separately from the version numbers used by the upstream CRA project.